sig-cli/annual-report-2021.md

```diff
@@ -127,11 +135,11 @@ Operational tasks in [sig-governance.md]:
       (or created if missing and your contributor steps and experience are different or more
       in-depth than the documentation listed in the general [contributor guide] and [devel] folder.)
```

Comment:
For the Readme, @eddiezane FYI the teams you're cleaning up in https://github.com/kubernetes/org/pull/3250/files are listed in it. And we have @soltysh PR open here: https://github.com/kubernetes/community/pull/6318. Otherwise I think it looks fine.

[Contributing.md](https://github.com/kubernetes/community/blob/master/sig-cli/CONTRIBUTING.md) though... it is out of date for sure, and also specific to kubectl. We also have separate/different [contributing docs on the SIG website](https://kubectl.docs.kubernetes.io/contributing/). I added an item to tomorrow's agenda about this.